src/core/infrastructure/repository/manager.ts

```diff
@@ -16,43 +28,52 @@ export const getGestionnaire = (demand: Demand): string | null => {
   return null;
 };
 
-export const getDemands = async (email: string): Promise<Demand[] | null> => {
-  const user = await getUser(email);
-
-  if (!user) {
-    return null;
-  }
-
-  const isDemoUser = user.gestionnaire === 'DEMO';
-  const records = await base(tableNameFcuDemands)
-    .select({
+export const getDemands = async (email?: string): Promise<Demand[] | null> => {
```

Comment:
Why did you change all of this ?

Reply:
All these changes are for call without email. In this case, the result is anonymized. (The personal data are not send)

Reply:
you don't need to complexify the code then... juste to consider people without email as demoUser (and change the name)

Reply:
Sure ? 
This is not exactly the same. 
For demo account, we send some fake value for personnal data.
For free account, we don't send personnal data.